handlers/BES_Modules_The_HDF5_Handler.adoc

```diff
@@ -143,30 +478,393 @@ H5.IgnoreUnknownTypes is true. See above.
+ care about the absolute path of object names. 
+ 
+H5.EnableFillValueCheck:::
+* When this key is set to __true__, the handler will check if the ___FillValue__ attribute holds the the correct datatype and the attribute value is inside the valid data range.
+
+* We find that occasionally that the datatype of attribute _FillValue is different 
+ than the datatype of the corresponding variable for some NASA HDF5 products. 
+ This violates the CF conventions. So the handler
+ corrects the FillValue datatype to make it the same as the corresponding variable datatype. However, the
+ original value of the ___fillvalue__ may also fall out of the range of the variable datatype. This can be illustrated with the following example.
+ ** The variable and the ___fillvalue__ are present as follows:
+ *** variable datatype: __unsigned char__  
+ *** _fillvalue attribute datatype: __signed char__
+ *** the value of the ___fillvalue__: -127
+ 
+ ** NOTE: the value of the ___filevalue__(-127) is out of the range of the __unsigned char__, which can not be a negative number. 
+ ** If such a case occurs, we believe this is a data producer's mistake and the hyrax service should return an error. The Hyrax data service center should report this issue back to the data producer. 
+ However, this may only occur for one or two variables and the data center may not 
+ want to stop the hyrax service. So we provide this BES key so that 
+ the data center can have an option to continue the service and may use NcML to patch the 
+ wrong fillvalue until the data producer corrects the wrong ___fillvalue__ in the new release.
+ **  By default, this key is set to __true__. If the fillvalue is out of the range of the variable type, 
+ Hyrax generates an error and the service stops. 
+ ** To ignore the ___fillvalue__ check, set this key to __false__. The service runs normally but
+ the ___Fillvalue__ of some variables may be wrong and it will cause issues on the client-side.
+
+H5.EnableDAP4Coverage:::
+* If this key is set to __true__, the handler adds the DAP4 coverage information to the DMR. By default, this key is set to __true__. 
+ 
+ H5.EnableCheckNameClashing:::
+ * When this key is set to __true__, the handler will check if there exists name clashing among variables and attributes. If name clashing occurs, the handler tries to resolve the name clashing by generating unique names for the clashed ones.
+ For NASA HDF5 and HDF-EOS5 products, we don't see any name clashings for variables and attributes. In fact, unlike HDF4, it is very rare to have name clashing for HDF5. So to reduce performance overhead, we set this key to __false__ by default. 
+ Users can set this key to __true__ if it becomes necessary.
+ 
+H5.NoZeroSizeFullnameAttr:::
+* When this key is set to __true__, the fullnamepath attribute will NOT be added if the HDF5 variable data storage size is 0. This is necessary to generate correct HDF5 DMRRPP files.
+
+H5.EscapeUTF8Attr:::
```

Comment:
Is `true` the default? I think maybe, for clarity these key descriptions could be written like:
```
H5.EscapeUTF8Attr:::
* default=true
* When this key is set to __true__, the attribute values that use UTF-8 character
...
```

Reply:
Like the approach. Will add it.